user_api_app/admin.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from .models import CustomUser
+
+
+class CustomUserAdmin(UserAdmin):
+    model = CustomUser
+    list_display = (
+        "email",
+        "age",
+        "country",
+        "is_staff",
+        "is_active",
+    )
+    list_filter = (
+        "email",
+        "age",
+        "country",
+        "is_staff",
+        "is_active",
+    )
+    fieldsets = (
+        (None, {"fields": ("email", "password", "age", "country")}),
+        (
+            "Permissions",
+            {"fields": ("is_staff", "is_active", "groups", "user_permissions")},
+        ),
+    )
+    add_fieldsets = (
+        (
+            None,
+            {
+                "classes": ("wide",),
+                "fields": (
+                    "email",
+                    "password1",
+                    "password2",
+                    "age",
+                    "country",
+                    "is_staff",
+                    "is_active",
+                    "groups",
+                    "user_permissions",
+                ),
+            },
+        ),
+    )
```

Comment:
Do we need to provide all these usually they are provided by Django itself

If we have to change anything we can inherit from Django's classes

Reply:
I have checked this feature, if i remove it then i am not even seeing custom users tab in the admin portal.

Reply:
Lets discuss this on call